.travis.yml

```diff
@@ -1,96 +1,126 @@
 language: python
 
+python:
+    - 2.7
+    - 3.4
+    - 3.5
+
 # Setting sudo to false opts in to Travis-CI container-based builds.
 sudo: false
 
-# The apt packages below are needed for sphinx builds, which can no longer
-# be installed with sudo apt-get.
+
+# The apt packages below are needed for sphinx builds. A full list of packages
+# that can be included can be found here:
+#
+# https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
+
 addons:
     apt:
         packages:
             - graphviz
             - texlive-latex-extra
             - dvipng
 
-python:
-    - 2.7
-    - 3.4
-    - 3.5
-
 env:
     global:
         # The following versions are the 'default' for tests, unless
         # overidden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
-        - NUMPY_VERSION=1.10
+        - NUMPY_VERSION=stable
         - ASTROPY_VERSION=stable
-        - CONDA_DEPENDENCIES='Cython scipy scikit-image matplotlib'
+        - SETUP_CMD='test'
         - PIP_DEPENDENCIES=''
+        - CONDA_DEPENDENCIES='Cython scipy scikit-image matplotlib jupyter'
 
     matrix:
-
         # Make sure that egg_info works without dependencies
         - SETUP_CMD='egg_info'
-
         # Try all python versions with the latest numpy
         - SETUP_CMD='test'
 
 matrix:
     include:
 
-        # Do a coverage test in Python 2. This requires the latest
-        # development version of Astropy, which fixes some issues with
-        # coverage testing in affiliated packages.
+        # Do a coverage test in Python 2.
         - python: 2.7
-          env: ASTROPY_VERSION=development SETUP_CMD='test --coverage'
+          env: SETUP_CMD='test --coverage'
 
         # Check for sphinx doc build warnings - we do this first because it
         # may run for a long time
         - python: 2.7
-          env: ASTROPY_VERSION=development SETUP_CMD='build_sphinx -w' CONDA_DEPENDENCIES='Cython scipy scikit-image matplotlib jupyter'
-        - python: 3.4
-          env: ASTROPY_VERSION=development SETUP_CMD='build_sphinx -w' CONDA_DEPENDENCIES='Cython scipy scikit-image jupyter'
```

Comment:
Don't mind me, it was a stupid question, ci-helpers installs it as default for the docs builds.